package.json

```diff
@@ -7,18 +7,21 @@
   "license": "GPL-3.0",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
     "start": "wp-scripts start",
     "build": "wp-scripts build --mode production",
     "lint:css": "wp-scripts lint-style",
-    "setupgithooks": "bin/copy-git-hooks.sh"
+    "setupgithooks": "bin/copy-git-hooks.sh",
+    "test": "env JEST_TIMEOUT=10000 WP_USERNAME=admin WP_PASSWORD=admin WP_BASE_URL=https://www.planet4.test wp-scripts test-e2e",
+    "test:watch": "env JEST_TIMEOUT=30000 PUPPETEER_HEADLESS=false PUPPETEER_SLOWMO=100 WP_USERNAME=admin WP_PASSWORD=admin WP_BASE_URL=https://www.planet4.test wp-scripts test-e2e"
   },
   "devDependencies": {
     "@wordpress/components": "^8.3.2",
     "@wordpress/data": "^4.9.2",
+    "@wordpress/e2e-test-utils": "^4.5.0",
```

Comment:
The latest version of the E2E test utils is 4.7.0 (https://www.npmjs.com/package/@wordpress/e2e-test-utils?activeTab=readme), but it had some differences with our setup, for example, it threw:
```
No node found for selector: .block-editor-inserter__search-input
``` 
When trying to use the `insertBlock` helper function, because the selector changed from one version to another. 

I found this message on the E2E docs:
```
It works properly with the minimum version of Gutenberg 5.6.0 or the minimum version of WordPress 5.2.0.
```
But I'm not sure what they mean with `Gutenberg 5.6.0` as Gutenberg is actually a set of packages, there is no package named `gutenberg` in NPM... the latest WP version is 5.4.1. So, I don't know what that `5.6.0` version refers to...

Reply:
Maybe they're talking about the version of the Gutenberg plugin? https://wordpress.org/plugins/gutenberg/